packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbarItemWithChipGroup.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { DataToolbarItem, DataToolbarItemProps } from './DataToolbarItem';
+import { ChipGroup, Chip, ChipGroupToolbarItem } from '../../../components/ChipGroup';
+import { DataToolbarContext } from './DataToolbarUtils';
+import styles from '@patternfly/react-styles/css/components/DataToolbar/data-toolbar';
+import { getModifier } from '@patternfly/react-styles';
+
+export interface DataToolbarItemWithChipGroupProps extends DataToolbarItemProps {
+  /** An array of strings to be displayed as chips in the expandable content */
+  chips?: string[];
+  /** Callback passed by consumer used to delete a chip from the chips[] */
+  deleteChip?: (chip: string) => void;
+  /** Content to be rendered inside the Data toolbar item associated with the chip group */
+  children: React.ReactNode;
+  /** Unique label for the chip group */
+  label: string;
```

Comment:
This should also be `React.ReactNode`.

Reply:
so, I renamed this prop to categoryName, because that is what it is - and categoryName is a prop on a chip group that must be a string.